contrib/mesos/pkg/scheduler/plugin.go

```diff
@@ -770,7 +770,7 @@ func (s *schedulingPlugin) scheduleOne() {
 	dest, err := s.config.Algorithm.Schedule(pod, s.config.MinionLister) // call kubeScheduler.Schedule
 	if err != nil {
 		log.V(1).Infof("Failed to schedule: %v", pod)
-		s.config.Recorder.Eventf(pod, "failedScheduling", "Error scheduling: %v", err)
+		s.config.Recorder.Eventf(pod, "FailedScheduling", "Error scheduling: %v", err)
```

Comment:
Do you have examples of systems interpreting the events? We've said that they shouldn't be treated as an API.